app/assets/stylesheets/admin/loans/timeline/_table.scss

```diff
@@ -42,4 +42,26 @@
     margin-left: 10px;
     color: $gray4;
   }
+
+  // Delete button enabled
+  .project-group .dropdown-menu {
```

Comment:
Let's be sure to reuse this stuff if we need to be able to disable the delete link for steps also.